docs/docs/configuration/env-variables.md

```diff
@@ -3,100 +3,84 @@
 This page contains all available environment variables supported by Mathesar. See the specific installation guides for the applicable environment variables and instructions on how to set them.
 
 
-## Backend Configuration {: #backend}
+## Backend configuration {: #backend}
 
-### `SECRET_KEY`
+### `SECRET KEY`
```

Comment:
Why is the underscore removed here?

Reply:
Typo

Reply:
Fixed in https://github.com/centerofci/mathesar/pull/2907/commits/4e7ddb6b6bb06399b350fef1613b9497b754ab42